.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Please remove this file.

Reply:
yes removed.Thank you for noticing

Reply:
Please accept this and close this issue #1236